tests/Integrations/PHPRedis/V5/PHPRedisClusterTest.php

```diff
@@ -528,10 +528,10 @@ public function dataProviderTestHashFunctions()
             ],
             [
                 'hIncrByFloat', // method
-                [ 'h1', 'k3', 1.6 ], // arguments
-                4.6, // expected result
-                [ 'k1' => 'v1', 'k2' => 'v2', 'k3' => '4.6' ], // expected final hash value
-                'hIncrByFloat h1 k3 1.6', // raw command
+                [ 'h1', 'k3', 1.5 ], // arguments
```

Comment:
ok now I am very curious about this. is it due to some rounding/comparison issues with float?

Reply:
correct, the newer version is more _precise_ in its output - and a 1.5 and 4.5 can be represented exactly unlike 1.6 and 4.6.